crates/ruff_cli/src/lib.rs

```diff
@@ -177,6 +177,11 @@ fn format(args: FormatCommand, log_level: LogLevel) -> Result<ExitStatus> {
 }
 
 pub fn check(args: CheckCommand, log_level: LogLevel) -> Result<ExitStatus> {
+    #[allow(deprecated)]
+    if args.line_length.is_some() {
+        warn_user_once!("The option `--line-length` has been renamed to `--line-width` to emphasize that the limit is the display width of a line and not the number of characters. Use the option `--line-width` instead.");
+    }
```

Comment:
Do you think it would be useful to specify _when_ the option will be removed in the deprecation message? Like "This option will be removed in v0.3.0". It could also help us in finding the deprecated features which needs to be removed before bumping the version.

Reply:
I don't think I have enough historical data yet to make a good prediction. Like removing it in 0.3 would be too early if we happen to release it in the coming weeks.